content/cloud/cloud-command-reference.md

```diff
@@ -14,7 +14,13 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-comman
 - [spin cloud apps list](#spin-cloud-apps-list)
 - [spin cloud deploy](#spin-cloud-deploy)
 - [spin cloud help](#spin-cloud-help)
+- [spin cloud key-value](#spin-cloud-key-value)
+- [spin cloud key-value create](#spin-cloud-key-value-create)
+- [spin cloud key-value delete](#spin-cloud-key-value-delete)
+- [spin cloud key-value list](#spin-cloud-key-value-list)
+- [spin cloud key-value rename](#spin-cloud-key-value-rename)
 - [spin cloud link](#spin-cloud-link)
+- [spin cloud link key-value-store](#spin-cloud-link-key-value-store)
 - [spin cloud link sqlite](#spin-cloud-link-sqlite)
```

Comment:
I believe we are missing `spin cloud key-value set`

Reply:
Good catch, thanks @kate-goldenring 
The new update/correction will be in the next commit/push